src/registrar/config/settings.py

```diff
@@ -285,7 +285,7 @@
 
 # Content-Security-Policy configuration
 # this can be restrictive because we have few external scripts
-allowed_sources = ("'self'",)
+allowed_sources = ("'self'")
```

Comment:
**suggestion**: The trailing comma is important here to make `allowed_sources` into a one-element tuple. As the [`django-csp` docs](https://django-csp.readthedocs.io/en/latest/configuration.html) say "Note when a setting requires a tuple or list. Since Python strings are iterable, you may get very strange policies and errors."

Reply:
```suggestion
allowed_sources = ("'self'",)
```

Reply:
Good catch, thanks!